### PR TITLE
Create modal to exit conversation

### PR DESCRIPTION
**Pull request by ericharper1**
To test:

- start a conversation
- excluding the final page, at any point in the conversation click the exit button to quit

### CODE REVIEW

File: simcon_project/conversation_templates/views/conversation.py [hunk 1]
```diff
@@ -215,3 +215,11 @@ def conversation_end(request, ct_response_id):
         'ct_node_responses': ct_node_responses
     })
     return render(request, t, ctx)
+
+
+def exit_conversation(request):
+    if request.is_ajax():
+        ctx = {}
```
Comment (rosasck) [inline simcon_project/conversation_templates/views/conversation.py:245]: should this stay empty?

File: simcon_project/templates/conversation/exit_conversation_modal.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<form method="GET" action="">
+    {% csrf_token %}
+    <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+        </button>
+    </div>
+    <div class="modal-body">
+        <p>Are you sure you want to exit the conversation?
+            Your progress will <strong>NOT</strong> be saved
```
Comment (rosasck) [inline simcon_project/templates/conversation/exit_conversation_modal.html:11]: strong Not

**Review by rosasck — APPROVED** [review]
Works solidly

**Review by kaigoos — APPROVED** [review]
Looks good!

**Review by Mkay971 — APPROVED** [review]

**Pull request merged**